phpBB/includes/acp/acp_bbcodes.php

```diff
@@ -269,6 +269,7 @@ function main($id, $mode)
 
```

Comment:
To what end? The BBCodeMonkey class is already used when `$configurator->BBCodes->addCustom()` is called during configuration. It could be used in the ACP to more accurately detect unsafe BBCodes or broken definitions but it would require a proper interface. If you have a suggestion, you can develop your idea in the A51 thread.